server/src/__tests__/fixtures/completion.bb

```diff
@@ -29,9 +29,11 @@ do_foo() {
     $D
 }
 
+LICENSE = "GPL-2.with-bisn-e"
```

Comment:
You could add another test case with an empty quote, like shown in my example, it currently fails so it's a good thing to test in the future!

Reply:
Good catch.

Fixed and tested.